src/UserService.Business/Commands/User/EditUserCommand.cs

```diff
@@ -1,105 +1,140 @@
+      _cacheNotebook = cacheNotebook;
+      _bus = bus;
+    }
+
+    /// <inheritdoc/>
+    public async Task<OperationResultResponse<bool>> ExecuteAsync(Guid userId, JsonPatchDocument<EditUserRequest> patch)
+    {
+      Guid requestSenderId = _httpContextAccessor.HttpContext.GetUserId();
+
+      bool isAdmin = await _accessValidator.IsAdminAsync(userId);
+      bool isAddEditRemoveUsers = await _accessValidator.HasRightsAsync(Rights.AddEditRemoveUsers);
+
+      if (!isAddEditRemoveUsers && userId != requestSenderId)
+      {
+        _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.Forbidden);
+      }
+
+      Operation<EditUserRequest> isActiveOperation = patch.Operations.FirstOrDefault(
+        o => o.path.EndsWith(nameof(EditUserRequest.IsActive), StringComparison.OrdinalIgnoreCase));
+
+      Operation<EditUserRequest> isAdminOperation = patch.Operations.FirstOrDefault(
+        o => o.path.EndsWith(nameof(EditUserRequest.IsAdmin), StringComparison.OrdinalIgnoreCase));
+
+      Operation<EditUserRequest> isGenderOperation = patch.Operations.FirstOrDefault(
+        o => o.path.EndsWith(nameof(EditUserRequest.GenderId), StringComparison.OrdinalIgnoreCase));
+
+      if ((!isAddEditRemoveUsers && userId != requestSenderId) ||
+        (isActiveOperation is not null && !isAddEditRemoveUsers) || 
+        (isAdminOperation is not null && !isAdmin) || 
+        (isGenderOperation is not null && userId != requestSenderId))
+      {
+        return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.Forbidden);
+      }
+
+      OperationResultResponse<bool> response = new();
+
+      if (isActiveOperation != null)
+      {
+        bool newValue = bool.Parse(isActiveOperation.value.ToString());
+
+        bool switchActiveStatusResult = await _credentialsRepository.SwitchActiveStatusAsync(userId, newValue);
+
+        if (!switchActiveStatusResult)
+        {
+          response.Errors.Add("Can not change is active status.");
+        }
+        else if (!newValue)
+        {
+          await _bus.Publish<IDisactivateUserRequest>(IDisactivateUserRequest.CreateObj(
+            userId,
+            requestSenderId));
+        }
+      }
+
+      (JsonPatchDocument<DbUser> dbUserPatch, JsonPatchDocument<DbUserAddition> dbUserAdditionPatch) = _mapperUser.Map(patch);
+
+      if (dbUserPatch.Operations.Any())
+      {
+        response.Body = await _userRepository.EditUserAsync(userId, dbUserPatch);
+
+        if (!response.Body)
+        {
+          return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.NotFound);
+        }
+      }
+
+      if (dbUserAdditionPatch.Operations.Any())
+      {
+        response.Body = await _userRepository.EditUserAdditionAsync(userId, dbUserAdditionPatch);
+
+        if (!response.Body)
+        {
+          return _responseCreator.CreateFailureResponse<bool>(HttpStatusCode.NotFound);
+        }
+      }
+
+      await _cacheNotebook.RemoveAsync(userId);
+

```

Comment:
anyway you get empty errors list.  remove 96-98 lines
something strange in 100 line, remove it